### PR TITLE
Libjpeg decompress target

### PR DESCRIPTION
**Pull request by Avgor46**

### CODE REVIEW

File: projects/libjpeg/decompress.toml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+[sydr]
+args = "-l debug -j2 -s 90"
```
Comment (SweetVishnya) [inline projects/libjpeg/decompress.toml:18]: we do not set -s anymore, now it is set automatically

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

**Comment by SweetVishnya** [comment]
Update fuzz targets in README

File: projects/libjpeg/decompress.toml (hunk 1 above)
Comment (Avgor46) [reply projects/libjpeg/decompress.toml]: done

File: projects/libjpeg/decompress.toml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+target = "/decompress_sydr @@"
+jobs = 2
+
+[libfuzzer]
+path = "/decompress_fuzzer"
+args = "-rss_limit_mb=0 /corpus_decompress"
```
[the remark this answers is not in the document]
Comment (Avgor46) [reply projects/libjpeg/decompress.toml:24]: with "true" it should work now

**Pull request merged**